org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/analysis/SlicingRunnableDevice.java

```diff
@@ -0,0 +1,101 @@
+package org.eclipse.scanning.sequencer.analysis;
+
+import java.util.Arrays;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Arrays'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)